### DISCUSSION TITLE
Create an `awesome-directus` list

### DISCUSSION THREAD

**Discussion by paescuj** [directus/directus#6102] [Feature Requests]
How about creating an [awesome-list](https://github.com/topics/awesome-list) for Directus?
This could be used to provide useful links like blog articles, extensions, projects built with Directus, etc.
E.g. https://github.com/gremo/react-directus

**Comment by benhaynes** [comment]
Could be a repo on the `community-directus` GitHub Org. Then everyone can submit new resources via PRs. @rijkvanzanten ?

**Reply by rijkvanzanten** [reply]
That's what I was gonna suggest 👍🏻 

@paescuj Do you wanna lead that effort? I can set you up with a repo 🙂

**Reply by paescuj** [reply]
Sure, I can come up with something 😄

**Reply by rijkvanzanten** [reply]
Sweet! Created [the repo](https://github.com/directus-community/awesome-directus) and sent you an invite 🙂

**Reply by paescuj** [reply]
Nice, thanks! Will work on it as soon as I have time 👍

**Comment by br41nslug** [comment]
This was created at https://github.com/directus-community/awesome-directus
